### PR TITLE
Remove duplicate code tidy integration tests

### PR DESCRIPTION
**Pull request by mapingo**
### Change description ###
Remove duplicate duplicate mini draft document removal code
Tidy up wiremock test utils for integration tests

### CODE REVIEW

File: ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/event/solicitor/page/SolAboutThePetitioner.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.divorce.ccd.event.solicitor.page;
+
+import uk.gov.hmcts.reform.divorce.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.reform.divorce.ccd.builder.FieldCollectionBuilderExtension;
+import uk.gov.hmcts.reform.divorce.ccd.model.CaseData;
+
+public class SolAboutThePetitioner implements CcdPageConfiguration {
```
Comment (mapingo) [inline ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/event/solicitor/page/SolAboutThePetitioner.java:7]: Incorrect class name

File: ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/event/solicitor/page/SolAboutTheRespondent.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.divorce.ccd.event.solicitor.page;
+
+import uk.gov.hmcts.reform.divorce.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.reform.divorce.ccd.builder.FieldCollectionBuilderExtension;
+import uk.gov.hmcts.reform.divorce.ccd.model.CaseData;
+
+import static uk.gov.hmcts.ccd.sdk.api.DisplayContext.Mandatory;
+
+public class SolAboutTheRespondent implements CcdPageConfiguration {
```
Comment (mapingo) [inline ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/event/solicitor/page/SolAboutTheRespondent.java:9]: Incorrect class name

File: src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorCreateTest.java [hunk 3]
```diff
@@ -155,9 +151,9 @@ void shouldValidateApplicant1SolicitorOrgAndReturnNoErrorsWhenSolicitorBelongsTo
     public void shouldValidateApplicant1SolicitorOrgAndReturnErrorWhenSolicitorDoesNotBelongsToSelectedOrg() throws Exception {
         when(serviceTokenGenerator.generate()).thenReturn(TEST_SERVICE_AUTH_TOKEN);
 
-        stubGetOrganisationEndpoint("TESTORG123", objectMapper);
+        prdOrganisationUtil.stubGetOrganisationEndpoint("TESTORG123");
```
Comment (nitinprabhu) [inline src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorCreateTest.java:154]: `documentAssemblyUtil`  `prdOrganisationUtil` can have  `stubForDocAssembly` methods static so you won't need to auto wire them ?

Comment (mapingo) [reply src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorCreateTest.java]: Reverted to static and removed the need for the ObjectMapper when stubbing

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**